docs/graphql/manual/hasura-cli/hasura_migrate_squash.rst

```diff
@@ -0,0 +1,55 @@
+.. _hasura_migrate_squash:
+
+Hasura CLI: hasura migrate squash
+---------------------------------
+
+(PREVIEW) Squash multiple migrations into a single one.
+
+Synopsis
+~~~~~~~~
+
+
+(PREVIEW) Squash multiple migrations leading upto the latest one into a single migration file
+
+::
+
+  hasura migrate squash [flags]
+
+Examples
+~~~~~~~~
+
+::
+
+    # NOTE: This command is in PREVIEW, correctness is not guaranteed and the usage may change.
+
+    # squash all migrations from version 123 to the latest one:
+    hasura migrate squash --from 123
+
+Options
+~~~~~~~
+
+::
+
+      --admin-secret string   admin secret for Hasura GraphQL engine
+      --delete-source         delete the source files after squashing without any confirmation
+      --endpoint string       http(s) endpoint for Hasura GraphQL engine
+      --from uint             start squashing form this version
```

Comment:
I'm confused by the word "version" here as well. Isn't this more snapshot language?